lib/frame.rb

```diff
@@ -0,0 +1,20 @@
+class Frame
+  attr_reader :first_roll, :second_roll, :strike, :spare
```

Comment:
Maybe just access these four values by calling the methods? This could further simplify the class